pkg/apis/core/validation/shoot_test.go

```diff
@@ -2337,21 +2337,6 @@ var _ = Describe("Shoot Validation Tests", func() {
 					}))
 				})
 
-				It("should forbid specifying the SecurityContextDeny admission plugin", func() {
```

Comment:
Don't we need to preserve the test case but use another forbidden admission plugin?